SIT.Manager/ViewModels/Settings/EftViewModel.cs

```diff
@@ -1,27 +1,46 @@
-﻿using CommunityToolkit.Mvvm.Input;
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using FluentAvalonia.UI.Controls;
 using SIT.Manager.Interfaces;
+using System.IO;
 using System.Threading.Tasks;
 
 namespace SIT.Manager.ViewModels.Settings;
 
 public partial class EftViewModel : SettingsViewModelBase
 {
     private readonly IBarNotificationService _barNotificationService;
+    private readonly IInstallerService _installerService;
     private readonly ILocalizationService _localizationService;
     private readonly IVersionService _versionService;
 
+    [ObservableProperty]
+    private string _bsgEftInstallPath;
+
+    [ObservableProperty]
+    private string _sitEftInstallPath;
+
     public IAsyncRelayCommand ChangeInstallLocationCommand { get; }
 
     public EftViewModel(IBarNotificationService barNotificationService,
+        IInstallerService installerService,
         IManagerConfigService configService,
         ILocalizationService localizationService,
         IPickerDialogService pickerDialogService,
         IVersionService versionService) : base(configService, pickerDialogService)
     {
         _barNotificationService = barNotificationService;
+        _installerService = installerService;
         _localizationService = localizationService;
         _versionService = versionService;
 
+        BsgEftInstallPath = Path.GetDirectoryName(_installerService.GetEFTInstallPath()) ?? "Unable to detect BSG EFT install...";
+        SitEftInstallPath = _configsService.Config.SitEftInstallPath;
+        if (string.IsNullOrEmpty(SitEftInstallPath))
+        {
+            SitEftInstallPath = "SIT install not found. Please point to existing install here, or install by clicking the install button on the left";
```

Comment:
Should this be localized?

Reply:
oh ye, there's probably a couple other places in this I might have missed 😓

Reply:
Should have got all of them now.